fast/stages/1-tenant-factory/variables-fast.tf

```diff
@@ -56,11 +56,14 @@ variable "custom_roles" {
   type = object({
     gcve_network_admin              = string
```

Comment:
Following the same logic, should also at least gcve_network_admin be optional?

Reply:
_I'm guessing this is a merge artifact_

Reply:
yep, we don't have a rule for custom roles only needed by specific stages atm, this is all going to change anyway once we move to new resman, resolvign this but let's revisit ASAP and define a rule